Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -188,6 +188,42 @@
         callback(value, key.protectedProcess());
 }
 
+RemoteLayerTreeDrawingAreaProxy::ProcessState& RemoteLayerTreeDrawingAreaProxy::processStateForIdentifier(WebCore::ProcessIdentifier identifier)
+{
+    RefPtr webProcess = WebProcessProxy::processForIdentifier(identifier);
+    RELEASE_ASSERT(webProcess);
```

Comment:
I think it would be better to not look up the webProcess at all.  We can just call pair.key.process().coreProcessIdentifier() and compare it with identifier below.  Same with the checking that it is the web page proxy's identifier.